package.json

```diff
@@ -7,6 +7,7 @@
   "private": true,
   "scripts": {
     "dev": "npm test && webpack-dev-server --config webpack/webpack.dev.js --open --hot --progress",
+    "dev:publish": "npm test && webpack-dev-server --config webpack/webpack.dev.js --open --hot --progress --host 0.0.0.0",
```

Comment:
"publish" isn't a great name for this. perhaps just make this what the "dev" target does?